.github/workflows/lint-test-chart.yaml

```diff
@@ -49,25 +40,50 @@ jobs:
           changed="$(ct list-changed)"
           if [[ -n "${changed}" ]]
           then
-            echo "changed=${{ toJSON(true) }}" >> $GITHUB_OUTPUT
+            echo "changed=true" >> "${GITHUB_OUTPUT}"
           else
-            echo "changed=${{ toJSON(false) }}" >> $GITHUB_OUTPUT
+            echo "changed=false" >> "${GITHUB_OUTPUT}"
           fi
 
+      - name: Get chart version
+        id: chart_version
+        if: steps.changes.outputs.changed == 'true'
+        uses: mikefarah/yq@dd648994340a5d03225d97abf19c9bf1086c3f07 # v4.40.5
+        with:
+          cmd: yq eval '.version' './charts/metrics-server/Chart.yaml'
+
+      - name: Get changelog entry
+        if: steps.changes.outputs.changed == 'true'
+        uses: mindsers/changelog-reader-action@b97ce03a10d9bdbb07beb491c76a5a01d78cd3ef # v2.2.2
+        with:
+          path: charts/metrics-server/CHANGELOG.md
+          version: ${{ steps.chart_version.outputs.version }}
```

Comment:
why do we do that in the lint-test-chart workflow?

Reply:
Because if the correct changelog entry doesn't exist and we don't check it during lint we get failures at release time which require more costly fixes.